.github/workflows/backport.yml

```diff
@@ -0,0 +1,16 @@
+name: Backport
+on:
+  pull_request:
+    types:
```

Comment:
Are all PRs merged into `main` branch needs to be backported?

Reply:
Not all of them. This means that the backport functionality gets triggered with every merge to main, but the PRs also need to be specially tagged for the actual backport to work (tagged as `backport-xxx` where `xxx` is the destination branch)

Reply:
I see. Added the link to the documentation in the PR description to provide more related information. Thanks @davidlago for the answer! 
Has this Github Action been tested?

Reply:
We're about to test it [here when this PR merges](https://github.com/opensearch-project/security/pull/1569)

Reply:
If I remember correctly, This new Github action won't take effective until this PR getting merged to `main` branch. 
Can we test it on the fork before merging? I told @nsri19 yesterday how to test CI/CD changes on fork for one of her PRs. It would be better to test it before merging to `main`

Reply:
For example, [this PR](https://github.com/opensearch-project/security/pull/1410) was tested with [this run](https://github.com/cliu123/security/actions/runs/1144958497) before merging to `main` branch.